lms/templates/courseware/courseware-chromeless.html

```diff
@@ -133,9 +133,27 @@
         var newWidth = contentElement.offsetWidth;
 
         if (eventType !== 'load' && newWidth === lastWidth && newHeight === lastHeight) {
+            // Check to see if the page is fully loaded
+            window.parent.postMessage("ready", document.referrer)
```

Comment:
How does this `postMessage()` check if the page is fully loaded? It seems like the call would just post a message and continue on? (I'm not suggesting this method is incorrect - I just truly don't know the answer...)

Reply:
Forgot to delete that line. That was an unnecessary postMessage that I was sending. The postMessage does not check that the document is ready it was just letting the parent page know that it was ready. I previously had `$(document).ready()` and that checked that the document was ready.

Reply:
@KristinAoki Nit: Just wanted to be sure you didn't miss my extra post-review comment above...